dockerfiles/theia-endpoint-runtime-binary/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia:${BUILD_TAG} as builder
+
+USER root
+
+RUN cd /home/theia && \
```

Comment:
it seems usually linters want that we use WORKDIR /home/theia before but I haven't run linter there so maybe it's fine